docs/usage/fields.md

```diff
@@ -0,0 +1,85 @@
+
+The `Field` class is used to define fields in models.
+
+TODO: Refactor the `Field customization` section in the `json_schema.md`.
```

Comment:
Assuming you want this PR merged before finishing this TODO, could you:
* add a note specifically saying to move documentation of `pydantic.fields.Field` here
* also mention that we should document all arguments to `Field` here, obviously that's closely related to the previous bullet — but not equivalent imo
* create a linear task for `usage/fields.md` and add it as a subtask of the `Review usage documentation` task

(The above bullets may or may not apply depending on how much work you want to put into this file before merging.)

Reply:
Hmm... Can't I just mention the 3 points that you said on the linear task? 🤔

Reply:
yes, as long as it is tracked somewhere that people will be forced to acknowledge